src/js/letters/containers/LetterList.jsx

```diff
@@ -57,7 +57,7 @@ export class LetterList extends React.Component {
           <div className="usa-alert-body">
             <h2 className="usa-alert-heading">Some letters may not be available</h2>
             <p className="usa-alert-text">
-              One of our systems appears to be down. If you believe you are missing a
+              One of our systems appears to be down. If you believe you‘re missing a
```

Comment:
Wrong curly quote. I think the one you want to use is `’`. Then again, it could be `’`... :man_shrugging: Unicode is hard.

Reply:
^ agree re:the type of curly quote it should be

Reply:
I mean, it needs to be updated to the correct apostrophe symbol :)

Reply:
Ugh, yep. ' vs. ‘ vs. ’ - had the second and needed the third. I seem to always forget that it's still an opening curly even though the shortcut uses a closing bracket. 👎